repository/esb-connector-stackexchange.properties

```diff
@@ -0,0 +1,28 @@
+# =========================
+# Properties related to App
+# =========================
+#
+# Your Access Token (Mandatory)
+accessToken=
+#
```

Comment:
can u put sample (dummy) value here? and for other properties as well

Reply:
Fix wiht d22b94f